src/roots.jl

```diff
@@ -138,32 +138,41 @@ function roots(f, X::IntervalLike{T}, ::Type{Bisection}, tol::Float64=1e-3; deri
     branch_and_prune(X, Bisection(f), tol)
 end
 
-function roots(f, X::Interval{T}, C::NewtonLike, tol::Float64=1e-3; deriv = nothing) where {T}
+function roots(f, X::Interval{T}, C::NewtonLike, tol::Float64=1e-3) where {T}
```

Comment:
You should probably just call the version of `roots` that has the explicit derivative argument here.

Reply:
Thanks for the comment; done, including some methods below.